tests/test_tftools.py

```diff
@@ -277,11 +277,11 @@ def test_placeholder_management():
     plh_mgr.add_placeholder('weights', tf.float64, sparse=True)
 
     # Get a dictionary of placeholders
-    plhs = plh_mgr.get_placeholders()
```

Comment:
Can you add a list placeholder and a list sparse placeholder to verify that it can make them and feed to them.